src/opnsense/service/templates/OPNsense/Proxy/parentproxy.conf

```diff
@@ -0,0 +1,27 @@
+{% if helpers.exists('OPNsense.proxy.general.parentproxy.enabled') and OPNsense.proxy.general.parentproxy.enabled == '1' %}
+{%   if helpers.exists('OPNsense.proxy.general.parentproxy.host') and OPNsense.proxy.general.parentproxy.host != '' %}
+{%     if helpers.exists('OPNsense.proxy.general.parentproxy.port') and OPNsense.proxy.general.parentproxy.port != '' %}
+cache_peer {{ OPNsense.proxy.general.parentproxy.host }} parent {{ OPNsense.proxy.general.parentproxy.port }} 0 no-query default
+{%     endif %}
+{%   endif %}
+{%   if helpers.exists('OPNsense.proxy.general.parentproxy.localdomains') and OPNsense.proxy.general.parentproxy.localdomains != '' %}
+acl ExcludePPDomains dstdomain {{ OPNsense.proxy.general.parentproxy.localdomains.replace(',', ' ') }}
+{%   endif %}
+{%   if helpers.exists('OPNsense.proxy.general.parentproxy.localips') and OPNsense.proxy.general.parentproxy.localips != '' %}
+acl ExcludePPIPs dst {{ OPNsense.proxy.general.parentproxy.localips.replace(',', ' ') }}
+{%   endif %}
+{%   if helpers.exists('OPNsense.proxy.general.parentproxy.localdomains') and OPNsense.proxy.general.parentproxy.localdomains != '' %}
+cache_peer_access {{ OPNsense.proxy.general.parentproxy.host }} deny ExcludePPDomains
+{%   endif %}
+{%   if helpers.exists('OPNsense.proxy.general.parentproxy.localips') and OPNsense.proxy.general.parentproxy.localips != '' %}
+cache_peer_access {{ OPNsense.proxy.general.parentproxy.host }} deny ExcludePPIPs
+{%   endif %}
+cache_peer_access {{ OPNsense.proxy.general.parentproxy.host }} allow all
+{%   if helpers.exists('OPNsense.proxy.general.parentproxy.localdomains') and OPNsense.proxy.general.parentproxy.localdomains != '' %}
+never_direct allow ExcludePPDomains
+{%   endif %}
+{%   if helpers.exists('OPNsense.proxy.general.parentproxy.localips') and OPNsense.proxy.general.parentproxy.localips != '' %}
```

Comment:
you may group the settings of `OPNsense.proxy.general.parentproxy.localips` together in a single if statement

Reply:
Wouldn't this mixup the order if acl, cache_peer_access and never_direct when putting in a single if? 
Config would look a bit weird.